### ISSUE TITLE
Cadl-ranch and test server cannot exist together

### ISSUE THREAD

**Issue by pshao25** [Azure/autorest.testserver#405]
cadl-ranch has `glob` dependency [8.0.3](https://github.com/Azure/cadl-ranch/blob/cc4dd0fbeb58e7e85fd5ec6282739c49338daa71/packages/cadl-ranch/package.json#L42), while test server now has `glob` dependency [7.2.3](https://github.com/Azure/autorest.testserver/blob/16bbd0ed8169a67f6bd5ff1b6f00f88a05778ca6/package-lock.json#L12091). There is a case we install both of them together and we might use version 8.0.3. Then we cannot successfully register the routes under `src\test-routes`.

Repro steps:
`npm install glob@8.0.3` under test server folder.
Start test server.

Symptom:
All the requests under `src\test-routes` get 404 response.

Root cause:
This [line](https://github.com/Azure/autorest.testserver/blob/16bbd0ed8169a67f6bd5ff1b6f00f88a05778ca6/src/app/app.ts#L350) cannot read files with `glob` version 8.0.3

**Comment by timotheeguerin** [comment]
Believe I fixed it by adding glob to the same version as cadl(not sure why it was not even a dependency) and normalizing the path (replace `\` with `/`) as seems like the library removed support for windows path

**Comment by antkmsft** [comment]
I think the issue should not be closed, but maybe renamed - it affects Windows + glob@8.0.3. It is not only `cadl-ranch` specific.
I was running `autorest.testserver` that I installed about a month ago. I updated it and it started giving me 404s.
`npm install glob@7.2.3` fixed it.
I think this should be fixed by, probably, normalizing paths.

`git clone` + `npm install` + `npm run start` should launch a testserver that works.
